apps/extension/src/core/util/urlToDomain.spec.ts

```diff
@@ -9,11 +9,10 @@ describe("Tests urlToDomain function", () => {
     expect(urlToDomain("https://something.com/anything?moreStuff=true&yes=more")).toEqual(
       Ok("something.com")
     )
-    expect(urlToDomain("Nope")).toEqual(Err("Invalid URL"))
-    expect(urlToDomain("7ca9d9e7-b68a-4d89-a4b1-a34d9766da3c.com")).toEqual(Err("Invalid URL"))
-    expect(urlToDomain("nope://something.com")).toEqual(Err("URL protocol unsupported"))
+    expect(urlToDomain("Nope").val).toEqual(Err(Errors.InvalidURL).val)
```

Comment:
These `Err` objects behave differently in the test environment under Jest 29, they must be compared on their `val` property rather than the instance itself.